utils/e2e-tests/bash/run-tests

```diff
@@ -14,7 +14,10 @@ export RUNTIME_WASM_PATH
 
 # Run all e2e tests.
 for TESTFILE in "$SCRIPT_DIR"/tests/"$TEST_SUITE"/*.sh; do
-  printf "=> Running test %s\n" "$TESTFILE" >&2
-  "$TESTFILE" "$@"
-  printf "=> Test %s passed\n" "$TESTFILE" >&2
+  # There are no tests for some test suits.
+  if [[ -f "$TESTFILE" ]]; then
+    printf "=> Running test %s\n" "$TESTFILE" >&2
+    "$TESTFILE" "$@"
+    printf "=> Test %s passed\n" "$TESTFILE" >&2
+  fi
```

Comment:
Shouldn't be possible... Am I missing somerthing?

Reply:
Have you tested it? Maybe I'm wrong here...
But what exactly is this test condition supposed to check? What the `TESTFILE` value will be?

Reply:
Sure, I tested it. 

First time, when I added `evm-tracing` test suite, I met an error that there are no files under `evm-tracing` to run the test. 

That's why I added this check to avoid having the error.

Reply:
Not relevant now